src/data/polynomial/ring_division.lean

```diff
@@ -144,14 +144,14 @@ have p = X - C (- p.coeff 0),
 this.symm ▸ prime_X_sub_C
 
 theorem irreducible_X_sub_C (r : R) : irreducible (X - C r) :=
-irreducible_of_prime prime_X_sub_C
+prime.irreducible prime_X_sub_C
 
 theorem irreducible_X : irreducible (X : polynomial R) :=
-irreducible_of_prime prime_X
+prime.irreducible prime_X
 
 lemma irreducible_of_degree_eq_one_of_monic (hp1 : degree p = 1)
```

Comment:
What about
```suggestion
lemma monic.irreducible_of_degree_eq_one (hp1 : degree p = 1)
```

Reply:
Sure, I'll do this one (and the `prime` above too), but for more of these changes, please open a new PR!